packages/cli/src/modules/mcp/tools/utils.ts

```diff
@@ -0,0 +1,145 @@
+						'\n\t - This webhook requires basic authentication with a username and password that should be provided by the user.';
+					break;
+				case 'headerAuth': {
+					const headerAuthDetails = await getHeaderAuthDetails(user, node, crednetialsService);
+					if (headerAuthDetails) {
+						credentialsInfo = headerAuthDetails;
+					}
+					break;
+				}
+				case 'jwtAuth': {
+					const jwtDetails = await getJWTAuthDetails(user, node, crednetialsService);
+					if (jwtDetails) {
+						credentialsInfo = jwtDetails;
+					}
+					break;
+				}
+			}
+		}
+
+		const responseModeDetails = getResponseModeDescription(node);
+		const pathParam = typeof node.parameters.path === 'string' ? node.parameters.path : '';
+		const httpMethod =
+			typeof node.parameters.httpMethod === 'string' ? node.parameters.httpMethod : 'GET';
+
+		return `
+				<trigger ${index + 1}>
+				\t - Node name: ${node.name}
+				\t - Base URL: ${baseUrl}
+				\t - PATH: ${isWorkflowActive ? '/webhook/' : '/webhook-test/'}${pathParam}
+				\t - HTTP Method: ${httpMethod}
+				\t - Response Mode: ${responseModeDetails}
+				${
+					credentialsInfo
+						? `\t - Credentials: ${credentialsInfo}`
+						: '\t - No credentials required for this webhook.'
+				}
+				</trigger ${index + 1}>`;
+	});
+
+	const webhookResults = await Promise.all(webhookPromises);
+	triggerNotice += webhookResults.join('\n\n');
+
+	return triggerNotice;
+};
+
+const getHeaderAuthDetails = async (
+	user: User,
+	node: INode,
+	credentialsService: CredentialsService,
+): Promise<string | null> => {
+	const id = node.credentials?.httpHeaderAuth?.id;
+	if (!id) return null;
+
+	const creds = await credentialsService.getOne(user, id, true);
+	if (hasHttpHeaderAuthDecryptedData(creds)) {
+		return `\n\t - This webhook requires a header with name "${creds.data.name}" and a value that should be provided by the user.`;
+	}
+	return null;
+};
+
+const getJWTAuthDetails = async (
+	user: User,
+	node: INode,
+	credentialsService: CredentialsService,
+): Promise<string | null> => {
+	const id = node.credentials?.jwtAuth?.id;
+	if (!id) return null;
+	try {
+		const creds = await credentialsService.getOne(user, id, true);
+		if (hasJwtSecretDecryptedData(creds)) {
+			return '\n\t - This webhook requires a JWT secret that should be provided by the user.';
+		} else if (hasJwtPemKeyDecryptedData(creds)) {
+			return '\n\t - This webhook requires JWT private and public keys that should be provided by the user.';
+		}
+	} catch {
+		return null;
+	}
+	return null;
+};
+
+const getResponseModeDescription = (node: INode): string => {
+	const responseMode =
+		typeof node.parameters.responseMode === 'string' ? node.parameters.responseMode : undefined;
+	let responseModeInfo =
+		'Webhook is configured to respond immediately with the message "Workflow got started."';
+	if (responseMode === 'responseNode') {
+		responseModeInfo = 'Webhook is configured to respond using "Respond to Webhook" node.';
```

Comment:
This function is really dense, making it difficult to read. Can we e.g. return from each condition separately? E.g. here 
```suggestion
		return 'Webhook is configured to respond using "Respond to Webhook" node.';
```